google-account-plugin/testSrc/com/google/gct/stats/KeyedExtensionUsageTrackerProviderTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2015 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.gct.stats;
+
+import static org.hamcrest.CoreMatchers.instanceOf;
+import com.intellij.testFramework.IdeaTestCase;
+import com.intellij.testFramework.LightPlatformTestCase;
+
+import junit.framework.TestCase;
+
+import org.junit.Assert;
+import org.junit.Ignore;
+
+/**
+ * Test to make sure we are loading the correct usage tracker when in the Cloud Tools plugin is in
+ * IntelliJ
+ */
+public class KeyedExtensionUsageTrackerProviderTest { // extends LightPlatformTestCase {
```

Comment:
I don't know how to. We can put off submitting until the test is ready. Or I can file a bug to do so.